### PR TITLE
Set Roboto as the default font family for Android

### PR DESCRIPTION
**Pull request by enahum**
#### Summary
In some Android devices (OnePlus and Oppo) the text is sometimes being cut-off and according to https://github.com/facebook/react-native/issues/15114 switching the default font to use Roboto fixes the issue

This is a great candidate to build the PR and have the reporter download the apk and test it.

Fixes: #2009 

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-11845

#### Device Information
This PR was tested on: Android Pie, Oreo and Nougat

### CODE REVIEW

File: app/app.js [hunk 1]
```diff
@@ -62,10 +62,37 @@ export default class App {
...
         Linking.addEventListener('url', this.handleDeepLink);
 
+        this.setFontFamily();
         this.getStartupThemes();
         this.getAppCredentials();
     }
 
+    setFontFamily = () => {
+        // Set a global font for Android
+        if (Platform.OS === 'android') {
```
Comment (hmhealey) [inline app/app.js]: Is it possible to do this from Android's styles.xml?

https://stackoverflow.com/questions/16404820/how-to-set-default-font-family-for-entire-android-app

File: ios/Mattermost.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -111,6 +110,18 @@
 		F006936FC2884C24A1321FC0 /* MaterialIcons.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 356C9186FA374641A00EB2EA /* MaterialIcons.ttf */; };
 		F083DB472349411A8E6E7AAD /* OpenSans-LightItalic.ttf in Resources */ = {isa = PBXBuildFile; fileRef = BE17F630DB5D41FD93F32D22 /* OpenSans-LightItalic.ttf */; };
 		F23C99AA5FA10E457A76803A /* libPods-MattermostTests.a in Frameworks */ = {isa = PBXBuildFile; fileRef = 4246DC09024BB33A3E491E25 /* libPods-MattermostTests.a */; };
+		DDE492F7425D451884DAA088 /* Roboto-Black.ttf in Resources */ = {isa = PBXBuildFile; fileRef = 6EFF13DD24CE4E26953E598A /* Roboto-Black.ttf */; };
```
Comment (hmhealey) [inline ios/Mattermost.xcodeproj/project.pbxproj]: Do we need to include these files on iOS even when we're not using the font?

File: app/app.js (hunk 1 above)
Comment (enahum) [reply app/app.js]: I was following this https://medium.com/react-native-training/react-native-custom-fonts-ccc9aacf9e5e and somehow it gives me more confidence that is going to use the right font

File: ios/Mattermost.xcodeproj/project.pbxproj (hunk 2 above)
Comment (enahum) [reply ios/Mattermost.xcodeproj/project.pbxproj]: this happens automatically when doing `react-native link` I don't see how this hurts at all

Comment (enahum) [reply ios/Mattermost.xcodeproj/project.pbxproj]: Yes we need the files

Comment (enahum) [reply ios/Mattermost.xcodeproj/project.pbxproj]: Thank you for that list, I’m going to remove the font files and see if the font displays on Android ;)

Comment (enahum) [reply ios/Mattermost.xcodeproj/project.pbxproj]: Ok offline decision is to leave the fonts as there might be some Android manufacturers who might not include the font in their devices.

**Comment by omorhefere** [comment]
How can I use this?

**Pull request merged**